algorithms/Python/number_theory/prime_number.py

```diff
@@ -0,0 +1,16 @@
+#TO CHECK WHETHER A NUMBER IS PRIME OR NOT
+
+
+N = int(input())
```

Comment:
Reading from stdin causes the python doctest to fail, it is preferred to provide sample tests in your program instead.

Reply:
I have done the requested changes. Thanks!